Frameworks/Foundation/NSDictionary.mm

```diff
@@ -99,8 +99,11 @@ - (NSArray*)allObjects {
         return nil;
     }
 
-    StrongId<NSArray> keyArray = [_keyEnumerator allObjects];
-    StrongId<NSMutableArray> toRet = [NSMutableArray new];
+    StrongId<NSArray> keyArray;
+    keyArray.attach([_keyEnumerator allObjects]);
+
+    StrongId<NSMutableArray> toRet;
```

Comment:
apologies. I mis-remembered what the copy attribute is signifying. Its copy-on-set not copy-on-get. I detest languages that use conventions as the primary mechanism to describe ownership.

---
In reply to: [73974316](https://github.com/Microsoft/WinObjC/pull/851#discussion_r73974316) [](ancestors = 73974316,73973850,73973752,73973293)